polymesh_schema.json

```diff
@@ -65,7 +65,7 @@
         "Permissions": {
             "asset": "Option<Vec<Ticker>>",
             "extrinsic": "Option<Vec<PalletPermissions>>",
-            "portfolio": "Option<Vec<PortfolioNumber>>"
+            "portfolio": "Option<Vec<PortfolioId>>"
```

Comment:
By typing this allows to permission secondary keys to access portfolios of identities other than the primary identity. Is this intended? Otherwise I think we should keep the identity out of this vector because all portfolios are of the same primary identity.

Reply:
This is intended. This feature can be used in combination with the custody feature. Alice can give custody of her portfolio to Bob and then bob can give permissions to control Alice's wallet to his secondary keys.

ps Without custody of a portfolio, this permission means nothing.

Reply:
Would it make sense to reflect that in the type? For example, `Option<Vec<PermissionedPortfolio>>` where
```
enum PermissionedPortfolio {
    Own(PortfolioNumber),
    Custodied(PortfolioId),
}
```
That might help to improve readability.

Reply:
That change to the type would be bad for performance though, if ever so slightly.

Reply:
Plus, it will have to be `Own(Option<PortfolioNumber>)` since the default portfolio can also be permissioned. Overall, I think it makes things more confusing and just using the standard PortfolioId everywhere is nicer.